<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html>
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>NGS tests</title>
<link rel="stylesheet" type="text/css" href="view.css" media="all">
<script type="text/javascript" src="view.js"></script>
<style>
.collapsible {
  background-color: white;
  color: black;
  cursor: pointer;
  padding: 8px;
  width: 100%;
  border: none;
  text-align: left;
  outline: none;
  font-size: 14px;
}

.active, .collapsible:hover {
  background-color: #fff2e6;
}

.content {
  padding: 0 18px;
  max-height: 0;
  overflow: hidden;
  transition: max-height 0.2s ease-out;
  background-color: #f1f1f1;
}
</style>
</head>
<body>


<div>

<img id="top" src="top.png" alt="">
<div id="form_container">
	<div class="form_description">
		<h2 align="center">
					Next Generation Sequencing
		</h2>
	</div>
<p>Next Generation Sequencing is ran on the Illumina MiniSeq and NextSeq 500, using "sequencing by synthesis" chemistry, upon different targeted sequencing panels.</p>
<p>
Genomic DNA QC : Quantified by Qubit and the integrity control by conventional PCR<br>
Library QC    : Check by automatic electrophoresis (Bioanalyser)<br>
Sequencing QC : Coverage > 500x, Q30 > 80%<br>
<br>
Bioinformatics :<br>
		<a href="https://support.illumina.com/help/BaseSpace_App_DNA_Amplicon_v2_OLH_1000000041403/Content/Source/HomePages/Home_Page_DNA_Amplicon_App.htm" target="_blank">
				DNA Amplicon (BaseSpace Workflow) 2.1.0.19 </a><br>
                via Illumina Local
                Run Manager on a desktop PC [DNA Amplicon Workflow (3.24.1.8+
                master), BWA-MEM Whole-Genome(Aligner)0.7.9a-isis-1.0.2, Pisces
                Variant Caller 5.2.9.23, Illumina Annotation Engine 2.0.11-0-
                g7fb24a09, Bam Metrics v0.0.22, and SAMtools 0.1.19-isis-1.0.3] 
  <br><br>
<img src=".\cap\Workflow_appDNAA_576x507.png" alt="workflow" width="600">
<br>
Curation      : <br>
Human GRCh37/hg19, BRCA Exchange, ClinVar, VarSome<br>
<br>
<br>
Limitations   : Our system could detect SNV-indel type  mutations with a variant allelic frequency greater or equal to 5%. Due to the nature of FFPE specimen and limitations of the amplicon-based library preparation, false negativity, including, but not limited to, large deletion, large duplication and gross chromosomal abnormalities, cannot be completely excluded.

</p>
<p align="center">
<img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAGAAAABgCAYAAADimHc4AAAABmJLR0QA/wD/AP+gvaeTAAAFzklEQVR4nO2dTWyURRiAn3e+3ZafgImG8JegiIZAhSKabowJxpMX1IvZYIwX8aonSf2JuhiV4omDBzxoDCYe9IRi4o3oSQgEKLT8SQIXEP8Ba2u7O68HCyJsW3a/mW9m4zzJHvr1m3feztOZeffbb3chkUgkEv9XJHQChfH24aVZRp8aKqqmImgPSB2xu+zyU69RrTZCpFUK0al3tp+Yhx19wJBVVKgI2gcsVQAFQSdPVFDpN2dWWguvhki182dAbW+JWQt6MrUVFSqI9qGsArIWovxgX+5d7CvF6ei8GTAwtCzTRkWNVsTSp8J6aMzVq/9KOm3r6IhbwMCB20pSftAqFVUqIvRBfZEKk6uHq47kY1eRWiUeAR8cKHOpvMZYKigVhD5gpVUMgPhZLH9D2WlXnHjdS/RbIPgekA0cedyizwnyGDDbY1d10CGEfWLZ17Cyn3tPHg9V/VwlnIBtQ/dkUv9QYYOP8IIigPnncXEskxVs6R3x0VcewixB7w4+ZKT+pcIdLsIJ/wz45GBf+/k6Fs6b+GvWFYhOgCm8x22H7jJGd5Nj8A1KhlJG6ULpxtKFUkIx6I2DD4Bm3WvzpO2LwmeAiNkJLLjl8ycfZnJw2/iPsQJnrOqc1pv6p9g9YGBwg0G/meaM31X4TlT2C439WaN+rpSVjrbQw2WQ06oMI3oQ5ODoWPkwtZ4/8qbui0JngBH7zBTFewPhJTs6fye15WN67SCUBg5dBBY2aXMB5SBwUGEoExn+Y6z3ODWxntL3QrFLkMrDTY+LbrX963Y0/ZXwLVYXqcggKoOqcmS0zLEYK5p2KHoPuLvZQTvBR1M1GOm/v+ovnfAUXQU1Xx7mlEcLziMaihWg/Nw0iYn6pkLziIhilyCRfaB33nRc2WEGBhda0U8ZnX+O2vKxQvMKSKFlaLb9yBOq7C6yzxk4j+gndrT8BrWe8RAJFLoENZaf/ErgQJF9zsASVPrN7Im3QiVQ7B5QrTYamdkMXCm035lQeTZU18VfC9qyZtCIPgn8XnjfEVK8AKDev26vzVivwtch+r8Z2RWs51AdX+O9I+tNg6dEeFRhWWZkScMW9sJu8E04vIAb2Hps3Ovov3lfV1R/c5AlKPEvSUBgkoDAJAGBSQICkwQEJgkITBIQmCQgMPHcnDsDW/ccb+n8Nzeu8pSJW9IMCEwSEJgkIDAdswd0ypreKmkGBCYJCEwSEJgkIDBJQGCSgMAkAYHpmOcB6VpQwgtJQGCSgMB0zB7QKWt6q8Q4A/7s0NhtEaOAVt6Y3SrHPMZui/gEiMdBEq9y2yI+AcpQR8ZukwgFiL8Z4DN2m0QnoJSVvA2Sz9jtEp2A11bLBWj+hu6c/DoZOyrifB4gDKE8cv2h3NeCItyAIcIZAIB6qIR8xHRArALcVysRVkAQqQAx7qsVHzFdEOUeUG6Ujo2bif8cy3stqGRKw7kCeCLKGfDKWvkN5Ly7iHL+1VXyi7t47ohSAIBgnS0ZLmO5JloBFndrtstYrolWgHFYtbiM5ZpoBVjjrm53Gcs10QqYWy8PMdWH/LWGhXKUFRBELGBLr4wAZx2EOlvrkWg/OTdaAQDiYPN0EcMnUQuwkr98dBHDJ1ELMOSvXlzE8EnUAqzNX724iOGTqAVguk4AEzOeNzX122d1nXSVjg+iFlDrkXHgdI4Qp168V/5ylY8PohYAoDleSMnTtiiiF2CMtL2J5mlbFNELsLb9MjJP26KIXgAlbX8Q87QtiOgFrD7afQahnS94GFt9tPuM84QcE72AalUaWFq7JwVAGa5WJejXFN4K0QsA2rqnR33e5OuQThEw3XePNcUIe32k4pqOEDCnXv4M4acWmvw4u17+3FtCDukIAVt6ZUSxzyPMvKYLDVWzefL1hOjpCAEAtZ5ZX4A+DVye8iThkqKbamtKe4rLLB9RfZT7rfDOsC6uN8ZfwMhGlBWTh79X0T1l6Xo/xjugE4lEIpFoxt9tzLjXTB5lDAAAAABJRU5ErkJggg=="/>
</p>


<button class="collapsible">Somatic and Germline BRCA1 and BRCA2 Testing (tumor tissue and blood)</button>
<div class="content">
<p>
Somatic mutation testing is done on the FFPE block or cytology cell block. <br><br>
Germline mutation testing is performed on FFPE block and peripheral white cells from the EDTA blood sample, and is also supplemented by MLPA.
</p>

<p>
Sequencing	  :	The AmpliSeq for Illumina BRCA Panel is a targeted resequencing 
				assay for somatic and germline variants across BRCA1 and BRCA2 genes, 
				targeting all the coding exonic regions and the flanking intronic sequences 
				of the BRCA1 and BRCA2 genes.
</p>
<p>
<a href=".\cap\MLPA_General_Protocol_MDP-v007.pdf" target="_blank">MLPA</a>          : SALSA MLPA Probemix kits (MRC, Holland):
                <a href=".\cap\Product_description_P002-D1_BRCA1-v08.pdf" target="_blank">P002 BRCA1</a>,
				<a href=".\cap\Product_description_P239-C1_BRCA1_region-v02.pdf" target="_blank">P090 BRCA2</a> and 
				<a href=".\cap\Product_description_P090-C1_BRCA2-v03.pdf" target="_blank">P239 BRCA1 region</a>;<br>
				confirmed by 
				<a href=".\cap\Product_description_P087-D1_BRCA1_Confirmation-v05.pdf"  target="_blank">P087 BRCA1 Confirmation</a> or
				<a href=".\cap\Product_description_P077-B1_BRCA2_Confirmation-v06.pdf"  target="_blank">P077 BRCA2 Confirmation</a> whichever is applicable;				
                <a href=".\cap\Coffalyser.Net-Reference-Manual-v07.pdf" target="_blank">Coffalyzer.Net software</a> v.210604.1451 (database v.210226.000) for <a href=".\cap\MLPA_QC_LKCHAN.pdf"  target="_blank">quality control</a>, visualization and analysis of data.

</p>
<p>
All findings are to be confirmed by orthogonal methods, including Sanger sequencing and others.
</p>
<p>
Consent is necessary for ordering this test (either somatic test alone, or somatic and germline tests). (<a href=".\cap\BRCA12Consent-e01.docx">Engish</a>) (<a href=".\cap\BRCA12Consent-c01.docx">Chinese</a>)
</p>
<p>
Pamphlets for BRCA testing in high staged, high grade ovarian cancer (<a href=".\cap\BRCA12Leaflet-e01.pdf" target="_blank">English</a>)(<a href=".\cap\BRCA12Leaflet-c01.pdf" target="_blank">Chinese</a>)
</p>
</div>
<button class="collapsible">Focus Panel for Solid Tumor (somatic DNA + RNA) [not yet launched]</button>
<div class="content">
<p>
The AmpliSeq for Illumina Focus Panel is a targeted resequencing assay for biomarker analysis of 52 genes with known relevance to solid tumors. All variants are to be confirmed by orthogonal methods (PCR-Sanger sequencing, RT-PCR + Sanger, FISH, etc.)
</p>
<p>
Hot spot mutations:<br>
AKT1, ALK, AR,	BRAF,	CDK4,	CTNNB1,	DDR2,	EGFR,	ERBB2,	ERBB3,	ERBB4,	ESR1,	FGFR1,	FGFR2,	FGFR3,	GNA11,	GNAQ,	HRAS,	IDH1,	IDH2,	JAK1,	JAK2,	JAK3,	KIT,	KRAS,	MAP2K1,	MAP2K2,	MET,	MTOR,	NRAS,	PDGFRA,	PIK3CA,	RAF1,	RET,	ROS1,	SMO
</p>

<p>
Copy Number Variants:<br>
ALK,	AR,	BRAF,	CCND1,	CDK4,	CDK6,	EGFR,	ERBB2,	FGFR1,	FGFR2,	FGFR3,	FGFR4,	KIT,	KRAS,	MET,	MYC,	MYCN,	PDGFRA,	PIK3CA
</p>

<p>
Fusion Drivers:<br>
	ABL1,	ALK,	AKT3,	AXL,	BRAF,	EGFR,	ERBB2,	ERG,	ETV1,	ETV4,	ETV5,	FGFR1,	FGFR2,	FGFR3,	MET,	NTRK1,	NTRK2,	NTRK3,	PDGFRA,	PPARG,	RAF1,	RET,	ROS1
<br>
<br>
<img src=".\cap\AmpliSeq_for_Illumina_Focus_Fusion_List.png" alt="Fusion list">
</p>
</div>

<button class="collapsible">Comprehensive Panel for Solild Tumor (somatic DNA + RNA)  [not yet launched]</button>
<div class="content">
<p>
The AmpliSeq for Illumina Comprehensive Panel v3 offers coverage of key cancer genes, including kinase domains and genes involved in DNA repair, targetting 161 unique cancer-associated genes
</p>
<p>
Hot spot mutations:<br>
AKT1, AKT2, AKT3, ALK, AR, ARAF, AXL, BRAF, BTK, CBL, CCND1, CDK4, CDK6, CHEK2, CSF1R, CTNNB1, DDR2, EGFR, ERBB2, ERBB3, ERBB4, ERCC2, ESR1, EZH2, FGFR1, FGFR2, FGFR3, FGFR4, FLT3, FOXL2, GATA2, GNA11, GNAQ, GNAS, H3F3A, HIST1H3B, HNF1A, HRAS, IDH1, IDH2, JAK1, JAK2, JAK3, KDR, KIT, KNSTRN, KRAS, MAGOH, MAP2K1, MAP2K2, MAP2K4, MAPK1, MAX, MDM4, MED12, MET, MTOR, MYC, MYCN, MYD88, NFE2L2, NRAS, NTRK1, NTRK2, PDGFRA, PDGFRB, PIK3CA, PIK3CB, PPP2R1A, PTPN11, RAC1, RAF1, RET, RHEB, RHOA, ROS1, SF3B1, SMAD4, SMO, SPOP, SRC, STAT3, TERT, TOP1, U2AF1, XPO1<br>
</p>
<p>
Full-length genes:<br>
ARID1A, ATM, ATR, ATRX, BAP1, BRCA1, BRCA2, CDK12, CDKN1B, CDKN2A, CDKN2B, CHEK1, CREBBP, FANCA, FANCD2, FANCI, FBXW7, MLH1, MRE11A, MSH2, MSH6, NBN, NF1, NF2, NOTCH1, NOTCH2, NOTCH3, PALB2, PIK3R1, PMS2, POLE, PTCH1, PTEN, RAD50, RAD51, RAD51B, RAD51C, RAD51D, RB1, RNF43, SETD2, SLX4, SMARCA4, SMARCB1, STK11, TP53, TSC1, TSC2<br>
</p>
<p>
Copy number variants:<br>
AKT1, AKT2, AKT3, ALK, AR, AXL, BRAF, CCND1, CCND2, CCND3, CCNE1, CDK2, CDK4, CDK6, CDKN2A, CDKN2B, EGFR, ERBB2, ESR1, FGF19, FGF3, FGFR1, FGFR2, FGFR3, FGFR4, FLT3, IGF1R, KIT, KRAS, MDM2, MDM4, MET, MYC, MYCL, MYCN, NTRK1, NTRK2, NTRK3, PDGFRA, PDGFRB, PIK3CA, PIK3CB, PPARG, RICTOR, TERT, TSC1, TSC2<br>
</p>
<p>
Fusion drivers:<br>
AKT2, ALK, AR, AXL, BRAF, BRCA1, BRCA2, CDKN2A, EGFR, ERBB2, ERBB4, ERG, ESR1, ETV1, ETV4, ETV5, FGFR1, FGFR2, FGFR3, FGR, FLT3, JAK2, KRAS, MDM4, MET, MYB, MYBL1, NF1, NOTCH1, NOTCH4, NRG1, NTRK1, NTRK2, NTRK3, NUTM1, PDGFRA, PDGFRB, PIK3CA, PPARG, PRKACA, PRKACB, PTEN, RAD51B, RAF1, RB1, RELA, RET, ROS1, RSPO2, RSPO3, TERT<br>
</p>
</div>
<script>
var coll = document.getElementsByClassName("collapsible");
var i;

for (i = 0; i < coll.length; i++) {
  coll[i].addEventListener("click", function() {
    this.classList.toggle("active");
    var content = this.nextElementSibling;
    if (content.style.maxHeight){
      content.style.maxHeight = null;
    } else {
      content.style.maxHeight = content.scrollHeight + "px";
    } 
  });
}
</script>
 
<br>
<div id = "footer">
Last upated on 02-Dec-2021.<br><br>
The html form and cascade style sheet were created on <a href="https://www.phpform.org" target="_blank">pForm</a>,<br>
with references to Javascripts and codes published in <a href="https://www.w3schools.com/" target="_blank">W3 School</a> and <a href="https://stackoverflow.com/" target="_blank">StackOverflow</a><br>
and image modified from <a href="https://icons8.com" target="_blank">Icons8</a>.
<br>
<br>
<br>
Anatomical Pathology Division, Queen Mary Hospital, Hong Kong.
</div>
</div>
<img id="bottom" src="bottom.png" alt="">
</div>


</body>
</html>
